src/components/form/form.jsx

```diff
@@ -17,7 +16,7 @@ const Form = React.forwardRef((props, forwardedRef) => {
   const [dirty, setDirty] = useState(false);
   const [valid, setValid] = useState(true);
 
-  const onDebouncedSubmit = useCallback(debounce(() => {
+  const onRegularSubmit = useCallback(() => {
```

Comment:
We can remove the `useCallback` since it's not doing anything useful in this usage.